BLSQ/iaso
# IA-1725: Completeness table - split forms counters

In the current situation, each row in the completeness table give 3 numbers (forms to fill, forms filled and percentage), and those values are aggregated so they count the forms/submissions linked to the current (row) OU, but also all of its descendants. 

This PR add two new columns to also show those values for forms/instances that are directly linked to the current OU (no descendants)

I also renamed the existing columns so they have a more clear/explicit name.

## Self proof reading checklist


## How to test

Go to the completeness table, make sure it still works and that the displayed counters make sense.

## Print screen / video

![image](https://user-images.githubusercontent.com/386387/206195096-f38a75bf-ea43-4c7a-8a3a-7ea7a74acd82.png)

## Notes

This PR is based on the branch of https://github.com/BLSQ/iaso/pull/233, so if that one could be merged first, that would make the code to review for this one shorter.
